Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityToolkitConfigurationProfileInspector.cs

```diff
@@ -114,7 +115,8 @@ public override void OnInspectorGUI()
                 EditorGUILayout.HelpBox("No Mixed Reality Toolkit found in scene.", MessageType.Warning);
                 if (GUILayout.Button("Click here to add Mixed Reality Toolkit instance to scene"))
                 {
-                    new GameObject("MixedRealityToolkit").AddComponent<MixedRealityToolkit>();
+                    MixedRealityInspectorUtility.AddMixedRealityToolkitToScene();
+                    EditorGUIUtility.PingObject(MixedRealityToolkit.Instance);
```

Comment:
Turns out this is superfluous because the process will be immediately hijacked by selecting the default config profile.